replication/binlogsyncer.go

```diff
@@ -76,6 +76,9 @@ type BinlogSyncerConfig struct {
 	// Use decimal.Decimal structure for decimals.
 	UseDecimal bool
 
+	// Use FloatWithTrailingZero structure for floats.
```

Comment:
Absolute nit: the go style guide recommends starting comments with the name of the element they document, so this comment should start with `FloatWithTrailingZero ...`